src/aura/BGE_EntryForm/BGE_EntryFormHelper.js

```diff
@@ -90,20 +90,34 @@
     },
 
     /**
-     * @description: queries open donations for upcoming donations
+     * @description: Gets open donations without any Payments and unpaid Payments
+     * for use in the Donation Selector component.
      * @return: void
      */
-    queryOpenDonations: function(component, donorId) {
+    getOpportunitiesWithOppPayments: function(component, recordId) {
         const donorType = component.get('v.donorType');
 
-        let action = component.get('c.getOpenDonations');
-        action.setParams({donorId: donorId, donorType: donorType});
+        let action = component.get('c.getOpportunitiesWithOppPayments');
+        action.setParams({donorId: recordId, donorType: donorType});
         action.setCallback(this, function (response) {
             const state = response.getState();
             if (state === 'SUCCESS') {
-                const openDonations = JSON.parse(response.getReturnValue());
-                component.set('v.openOpportunities', openDonations.openOpportunities);
-                component.set('v.unpaidPayments', openDonations.unpaidPayments);
+                const opportunitiesWithPayments = JSON.parse(response.getReturnValue());
+                let openOpportunitiesWithoutAnyOpenPayments = [];
+                let unpaidPayments = [];
+                opportunitiesWithPayments.forEach(function (opportunityWrapper) {
+                        if (opportunityWrapper.hasPayments === false ||
```

Comment:
Quick question around whether or not we put symbols in the next break when lines are getting long. Do we have a separate javascript style guide like the[ apex style guide](https://salesforce.quip.com/TkjIACNQIURb)? Did a quick search in quip and didn't come up with anything. Do we potentially have that somewhere else?

@ptewson-sfdo @kenlewis-sfdo

Reply:
@psadie great question - I think we we use the Google guidance on java style, but I might be conflating with a past project. That said, that's a great starting point: http://google.github.io/styleguide/jsguide.html

From that resource, "When a line is broken at an operator the break comes after the symbol" (so like Ken has it here). Which conveniently aligns with my own inclination, although I've heard arguments for putting boolean operators on the start of the newline for long conditionals in if statements that made some sense.